### PR TITLE
Added the Dusting API, put it into use and fixed some duster recipes.

### PR DESCRIPTION
**Pull request by iTitus**
Any more questions?
Feel free to ask.

### CODE REVIEW

File: src/main/java/buildcraftAdditions/BuildcraftAdditions.java [hunk 1]
```diff
@@ -64,19 +65,26 @@ public void preInit(FMLPreInitializationEvent event) {
...
-		DusterRecipes.dusting().addDusterRecipe(new ItemStack(Blocks.quartz_ore), new ItemStack(Items.quartz, 2));
-		DusterRecipes.dusting().addDusterRecipe(new ItemStack(Blocks.stone), new ItemStack(Blocks.gravel));
-		DusterRecipes.dusting().addDusterRecipe(new ItemStack(Blocks.cobblestone), new ItemStack(Blocks.sand));
-		DusterRecipes.dusting().addDusterRecipe(new ItemStack(Blocks.diamond_ore), new ItemStack(itemDust, 4));
-		DusterRecipes.dusting().addDusterRecipe(new ItemStack(Items.diamond), new ItemStack(itemDust, 2));
-		DusterRecipes.dusting().addDusterRecipe(Items.blaze_rod, new ItemStack(Items.blaze_powder, 4));
+		Item dustDiamond = new ItemDust(0x13ECFC).setUnlocalizedName("dustDiamond");
+		GameRegistry.registerItem(dustDiamond, "dustDiamond");
+		OreDictionary.registerOre("dustDiamond", dustDiamond);
+		Item dustEmerald = new ItemDust(0x17DD62).setUnlocalizedName("dustEmerald");
```
Comment (AEnterprise) [inline src/main/java/buildcraftAdditions/BuildcraftAdditions.java]: what's the user for the emerald dust?

**Comment by AEnterprise** [comment]
looking good but did you test and make sure that the code for things like removing a recipe also works as it should?

**Comment by iTitus** [comment]
Yeah I tested it. But I did not test the MineTweaker thing.
I should have added log error messages if a recipe is added, which is already there.

File: src/main/java/buildcraftAdditions/BuildcraftAdditions.java (hunk 1 above)
Comment (iTitus) [inline src/main/java/buildcraftAdditions/BuildcraftAdditions.java]: What is the use for diamond dust?

Comment (AEnterprise) [inline src/main/java/buildcraftAdditions/BuildcraftAdditions.java]: making kinetic coils and refinery plating

**Pull request merged**